conf/Makefile.chibios

```diff
@@ -95,6 +105,16 @@ ifeq ($(USE_LINK_GC),)
   USE_LINK_GC = yes
 endif
 
+# Linker extra options here.
+ifeq ($(USE_LDOPT),)
+  USE_LDOPT = 
```

Comment:
This is because it is based on original ChibiOS examples, makes it easier to upgrade afterwards. Should I still change it ?